frontend/src/ui/index.tsx

```diff
@@ -126,12 +128,21 @@ export const Title: React.FC<TitleProps> = ({ title, className }) => (
     <h2 className={className} css={{ margin: "16px 0" }}>{title}</h2>
 );
 
-export const FOCUS_STYLE_INSET = {
+/**
+ * Applies focus outline with a default width of 2.5px to elements.
+ * This should always be used instead of a custom focus property to ensure
+ * consistency throughout the design.
+ * If ({ inset: true }) is declared, the focus will be on the inside
```

Comment:
```suggestion
 * If `({ inset: true })` is declared, the focus will be on the inside
```

These docs support markdown.